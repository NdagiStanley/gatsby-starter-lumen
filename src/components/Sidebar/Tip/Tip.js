import React from 'react';

const Tip = () => (
  <div>
    <script type='text/javascript' src='https://storage.ko-fi.com/cdn/widget/Widget_2.js'></script>
    <script type='text/javascript'>kofiwidget2.init('Tip Me', '#008080', 'M4M5OEAR');kofiwidget2.draw();</script>
  </div>
);

export default Tip;
