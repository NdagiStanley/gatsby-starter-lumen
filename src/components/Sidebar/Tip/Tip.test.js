import React from 'react';
import renderer from 'react-test-renderer';
import Tip from './Tip';

describe('Tip', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Tip />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
