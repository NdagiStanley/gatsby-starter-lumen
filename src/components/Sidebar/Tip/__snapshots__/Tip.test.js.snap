// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tip renders correctly 1`] = `
<div>
  <script type='text/javascript' src='https://storage.ko-fi.com/cdn/widget/Widget_2.js'></script>
  <script type='text/javascript'>kofiwidget2.init('Tip Me', '#008080', 'M4M5OEAR');kofiwidget2.draw();</script>
</div>
`;
